<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>arrow-270.png</normaloff>arrow-270.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);

</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>61</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>cursive</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 136, 229);
font-family: &quot;Times New Roman&quot;, Times, serif;
font-family: cursive;</string>
   </property>
   <property name="text">
    <string>StockPro</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial,Helvetica,sans-serif</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 136, 229);
font-family: Arial, Helvetica, sans-serif</string>
   </property>
   <property name="text">
    <string>Stock per Material</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>661</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial,Lucida Console,monospace</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Arial&quot;, &quot;Lucida Console&quot;, monospace;</string>
   </property>
   <property name="text">
    <string>Batch number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="batchInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border: 1px solid #ced4da;
	border-radius: 10px;

}

QLineEdit:focus{
	border: 2px solid rgb(194, 219, 254);
}

QLineEdit:hover{
	border: 1px solid rgb(85, 170, 255);
}

	
</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Enter number</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scanButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	box-shadow:inset 0px 1px 0px 0px #ffffff;
	background:linear-gradient(to bottom, #ededed 5%, #dfdfdf 100%);
	background-color:#ededed;
	border-radius:8px;
	border:1px solid #dcdcdc;
	display:inline-block;
	cursor:pointer;
	color:#777777;
	font-family:Arial;
	font-size:12px;
	font-weight:bold;
	
	text-decoration:none;
	text-shadow:0px 1px 0px #ffffff;	

}

QPushButton:hover{
	background:linear-gradient(to bottom, #1E88E5 5%, #ffffff 100%);
	background-color:#1E88E5;
	color:rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string> Scan</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>camera--plus.png</normaloff>camera--plus.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial,Lucida Console,monospace</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Arial&quot;, &quot;Lucida Console&quot;, monospace;</string>
   </property>
   <property name="text">
    <string>Product ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="productInput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	border: 1px solid #ced4da;
	border-radius: 10px;

}

QLineEdit:focus{
	border: 2px solid rgb(194, 219, 254);
}

QLineEdit:hover{
	border: 1px solid rgb(85, 170, 255);
}


</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Enter ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial,Lucida Console,monospace</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Arial&quot;, &quot;Lucida Console&quot;, monospace;</string>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeDropdown">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	border: 1px solid #ced4da;
	border-radius: 8px;
	
}

QComboBox:hover{
	border: 1px solid rgb(85, 170, 255);
}

QComboBox::drop-down{
	border: 0px;
}

QComboBox::down-arrow{
	image: url(:/icon/down arrow.ico);
	width: 12px;
	height: 12px;
	margin-right: 15px;
}

/*style for QcomboBox after open select menu*/
QComboBox:on{
	border: 2px solid #c2dbfe;
}

/*style for list menu*/
QComboBox QListView{
	font-size: 12px;
	border: 1px solid rgba(0, 0, 0, 10%);
	padding: 5px;
	background-color: #fff;
	outline: 0px;
}

/*style for list of items*/
QComboBox QListView::item{
	padding-left: 10px;
	background-color: rgb(194, 219, 254);
}

QComboBox QListView::item:hover{
	background-color: rgb(194, 219, 254)
	
}




</string>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="placeholderText" stdset="0">
    <string/>
   </property>
   <item>
    <property name="text">
     <string>Appliances</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Computer Peripherals</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Computer Systems</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Electronics </string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gaming</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Health &amp; Sports</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Networking</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Software &amp; Services</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Smart Home &amp; Security</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Others</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>520</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	box-shadow:inset 0px 1px 0px 0px #ffffff;
	background:linear-gradient(to bottom, #ededed 5%, #dfdfdf 100%);
	background-color:#ededed;
	border-radius:6px;
	border:1px solid #dcdcdc;
	display:inline-block;
	cursor:pointer;
	color:#777777;
	font-family:Arial;
	font-size:12px;
	font-weight:bold;
	padding:6px 24px;
	text-decoration:none;
	text-shadow:0px 1px 0px #ffffff;
	

}

QPushButton:hover{
	background:linear-gradient(to bottom, #1E88E5 5%, #ffffff 100%);
	background-color:#1E88E5;
	color:rgb(255, 255, 255);
}


</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>520</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	box-shadow:inset 0px 1px 0px 0px #ffffff;
	background:linear-gradient(to bottom, #ededed 5%, #dfdfdf 100%);
	background-color:#ededed;
	border-radius:6px;
	border:1px solid #dcdcdc;
	display:inline-block;
	cursor:pointer;
	color:#777777;
	font-family:Arial;
	font-size:12px;
	font-weight:bold;
	padding:6px 24px;
	text-decoration:none;
	text-shadow:0px 1px 0px #ffffff;
	
}

QPushButton:hover{
	background:linear-gradient(to bottom, #1E88E5 5%, #ffffff 100%);
	background-color:#1E88E5;
	color:rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../icons/source.qrc"/>
 </resources>
 <connections/>
</ui>
